lib/CMakeLists.txt

```diff
@@ -44,124 +38,173 @@ add_library(arango_static_strings STATIC
   Basics/StaticStrings.cpp
 )
 
+set(FROZEN_INCLUDE_DIR "${PROJECT_SOURCE_DIR}/3rdParty/iresearch/external/frozen/include")
+set(FUNCTION2_INCLUDE_DIR "${PROJECT_SOURCE_DIR}/3rdParty/iresearch/external/function2")
+
+add_library(arango_lightweight STATIC
+        ApplicationFeatures/ApplicationFeature.cpp
+        ApplicationFeatures/ApplicationServer.cpp
+        ApplicationFeatures/ShellColorsFeature.cpp
+        Basics/ArangoGlobalContext.cpp
+        Basics/CleanupFunctions.cpp
+        Basics/ConditionLocker.cpp
+        Basics/ConditionVariable.cpp
+        Basics/ErrorCode.cpp
+        Basics/Exceptions.cpp Basics/Exceptions.tpp Basics/Exceptions.h
```

Comment:
It's strange, I think better to remove headers from this list or list them all, now it's mixed